Lib/statistics.py

```diff
@@ -304,6 +304,19 @@ def _fail_neg(values, errmsg='negative value'):
             raise StatisticsError(errmsg)
         yield x
 
+def _isqrt_frac_rto(n: int, m: int) -> float:
+    'Square root of n/m, rounded to the nearest integer using round-to-odd.'
+    a = math.isqrt(n*m) // m
```

Comment:
Sigh. This is still more complicated and inefficient than necessary. All we need is this:

```suggestion
    a = math.isqrt(n // m)
```

The goal is to assign `⌊√(n/m)⌋`  to `a`. But it's not hard to prove that `⌊√(n/m)⌋ = ⌊√(⌊n/m⌋)⌋`, which we can express in Python as just `math.isqrt(n // m)`.